### PR TITLE
chore(address): fix conversions 

### PR DESCRIPTION
**Pull request by calbera**

### CODE REVIEW

**Review by itsdevbear — APPROVED** [review]

File: cosmos/lib/conversions.go [hunk 1]
```diff
@@ -167,18 +168,22 @@ func SdkREToStakingRE(re []stakingtypes.RedelegationEntry) []staking.IStakingMod
...
+		}
 		pubKey, err := val.ConsPubKey()
 		if err != nil {
 			return nil, err
 		}
 		valsOut[i] = staking.IStakingModuleValidator{
-			OperatorAddress: val.OperatorAddress,
-			ConsensusPubkey: pubKey.Bytes(),
+			OperatorAddr:    operEthAddr,
+			ConsAddr:        pubKey.Address(),
```
Comment (calbera) [inline cosmos/lib/conversions.go:186]: @itsdevbear which of the 2 (`.Address()` or `.Bytes()`) to use here
image

Comment (calbera) [reply cosmos/lib/conversions.go:186]: actually I guess it wouldnt even be using this pubkey but the one in comet

Comment (calbera) [reply cosmos/lib/conversions.go:186]: https://github.com/cometbft/cometbft/blob/main/crypto/sr25519/pubkey.go#L31 Address() is correct, it is already truncated to 20 bytes

Comment (calbera) [reply cosmos/lib/conversions.go:186]: https://github.com/cometbft/cometbft/blob/main/crypto/tmhash/hash.go#L64

**Comment by itsdevbear** [comment]
i dont like but lgtm

**Pull request merged**